### ISSUE TITLE
[Question] Get all, without @paginate  and without custom resolver

### ISSUE THREAD

**Issue by robsontenorio** [nuwave/lighthouse#187]
Cant find any info on docs or issues. 

I can do this, great!

```
type Query{
   users: [User] @paginate(model: "App\\User")
}
```
But, the `count` parameter is required

```
{
  users(count: 10){
    data{
      id
      nome
    }
  }
}
```

I would like simple get all, without paginate. 

```
type Query{
   users: [User] 
}
```

But  ...
```
{
  "data": {
    "users": null
  },
  "errors": [
    {
      "message": "Class 'App\\Http\\GraphQL\\Queries\\User' not found",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ]
    }
  ]
}
```

I was wondering if exists some directive to simple get all, like `@find` or `@first`. Maybe `@all`, and a custom resolver wont be required.

And ...

```
type Query{
   users: [User] @all <-----
}
```

it would allow to query simple like this

```
{
  users{
      id
      name    
  }
}
```

**Comment by robsontenorio** [comment]
I have tried, but no lucky. Actually does not make sense.

```
type Query{
  users: [User] @hasMany
}
```

```
Call to a member function getKey() on null
```

**Comment by spawnia** [comment]
Closed in favour of #173 

We plan on implementing this, should be in one of the next versions of Lighthouse.
